app/controllers/dashboard_controller.rb

```diff
@@ -40,12 +40,19 @@ class DashboardController < ApplicationController
 
   def index
     @projects = @client.projects
+    if session[:last_project] && @projects && !@projects.empty?
+      curr_proj = @client.project(session[:last_project])
+      @projects.unshift @projects.delete(curr_proj)
+    end
+    if @projects && !@projects.empty?
```

Comment:
I'm not sure if this if statement is necessary.  The only scenario I can think of that this might be useful is when students have created a new project.

Reply:
This is the get the tests to pass - especially the day zero tests.